web/src/api/mqtt.jsx

```diff
@@ -34,7 +34,7 @@ export function MqttProvider({
   config,
   children,
   createWebsocket = defaultCreateWebsocket,
-  mqttUrl = `${baseUrl.replace(/^https?:/, 'ws:')}/ws`,
+  mqttUrl = `${baseUrl.replace(/^http/, 'ws')}/ws`,
```

Comment:
@paularmstrong this works. Let me know if you see a problem with this.

Reply:
Nope, no problem!

This should also be better for the HA addon. I haven't been able to test that yet. Do you want to publish this as a beta first?